### PR TITLE
Updated Coseismic and Secular NISAR Validation notebooks

### PR DESCRIPTION
**Pull request by mgovorcin**

### CODE REVIEW

File: methods/coseismic/Coseismic_Requirement_Validation-Ridgecrest_asc.ipynb [hunk 1]
```diff
@@ -6,45 +6,37 @@
     "tags": []
```
Comment (dbekaert) [inline methods/coseismic/Coseismic_Requirement_Validation-Ridgecrest_asc.ipynb:6]: <p>Can you change B to "Data Staging" header</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

**Comment by dbekaert** [comment]
Reached out to Adrian and CC-ed you to confirm some of the parameters.

**Comment by mgovorcin** [comment]
<p>yes, I will set it False</p>

---

 <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/22/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: methods/coseismic/Coseismic_Requirement_Validation-Ridgecrest_asc.ipynb (hunk 1 above)
Comment (dbekaert) [reply methods/coseismic/Coseismic_Requirement_Validation-Ridgecrest_asc.ipynb:6]: Can you do it in both those notebooks.

**Comment by mgovorcin** [comment]
<p>this changes when I change the ts end date to 20190101, together with other results</p>

---

 <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/22/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: methods/coseismic/Coseismic_Requirement_Validation-Ridgecrest_asc.ipynb (hunk 1 above)
Comment (mgovorcin) [reply methods/coseismic/Coseismic_Requirement_Validation-Ridgecrest_asc.ipynb]: <p>DONE</p>

**Review by dbekaert — APPROVED** [review]
Thanks @mgovorcin Looks good.

**Pull request merged**